packages/safe-tools-client/app/components/token-to-usd/index.gts

```diff
@@ -23,22 +25,23 @@ export default class TokenToUsd extends Component<Signature> {
 
   constructor(owner: unknown, args: Args) {
     super(owner, args);
-
     this.updateInterval = setInterval(() => {
       taskFor(this.tokenToUsdService.updateUsdConverter).perform(args.tokenAddress); 
     }, INTERVAL);
   }
   
   get usdAmount() {
-    return this.tokenToUsdService.toUsd(this.args.tokenAddress, BigNumber.from(this.args.tokenAmount))?.toString();
+    return this.tokenToUsdService.toUsd(this.args.tokenAddress, BigNumber.from(this.args.tokenAmount));
   }
 
   willDestroy() {
     clearInterval(this.updateInterval);
   }
 
   <template>
-    {{this.usdAmount}}
+    {{#if this.usdAmount}}
+      $ {{(weiToDecimal this.usdAmount @tokenDecimals)}} USD
+    {{/if}}
   </template>
 }
```

Comment:
Unrelated to this PR, but I think this might be better as a helper than a component. What do you think?

Reply:
Hmm, I think this should be possible. Let's go ahead with this PR and we can consider revisiting the possible conversion to a helper separately.